mlir/lib/Tools/mlir-pdll-lsp-server/PDLLServer.cpp

```diff
@@ -1621,7 +1622,8 @@ PDLTextFile::getPDLLViewOutput(lsp::PDLLViewOutputKind kind) {
         [&](PDLTextFileChunk &chunk) {
           chunk.document.getPDLLViewOutput(outputOS, kind);
         },
-        [&] { outputOS << "\n// -----\n\n"; });
```

Comment:
Note that I am not sure whether this and the other occurrence of the constant in the same file below *have* to be the same as in `mlir-opt` and `mlir-translate` or whether it this constant is local to the PDLL server. In the latter case, it might be better not to use the constant from `ToolUtilities.h`. What do you think?

Reply:
This is fine..